charts/cloudzero-agent/docs/sizing-guide.md

```diff
@@ -0,0 +1,104 @@
+### Prometheus Agent Deployment Sizing Guide
+
+#### Introduction
+
+This guide outlines the recommended settings for deploying the Prometheus agent using the CloudZero Helm chart. It includes instructions on how to configure memory and CPU resources based on the size of your cluster.
+
+#### Baseline Memory Requirements
+
+- **Base Memory:** 512Mi
+- **Base Memory Limit:** 1024Mi
+- **Additional Memory:** 0.75Gi per 100 nodes in the cluster
+
+#### Sizing Calculation
+
+It is recommended to consider the shape and size of your prometheus cluster when setting resource memory limits for the prometheus agent. To calculate the memory requirements for your cluster, one can use the following formula:
+
+![sizing formula](./assets/sizing-formula.png)
+
+> This guide uses a simple formula based on number of nodes, however your milage may vary if you have:
+> 
+> * Very large machines, with a large number of pods
+> 
+> * High churn pods or jobs. Each pod started will trigger allocation of a time seriese based metrics cache in the agent. If it restarts, a new cache is created. The cache will resize in memory for the look back period to handle remote write failures (2 hours).
+> 
+
+
+#### Sample values.yaml Configuration
+
+Modify or create a `values.yaml` file with the following content to configure the resource limits and requests for your Prometheus agent deployment. Replace `<NUMBER_OF_NODES>` with the actual number of nodes in your cluster:
```

Comment:
nit: I don't think `values.yaml` is what users would typically name yaml files that hold configuration for a given helm release. not that it wouldn't work, I just wouldn't want to confuse users with the default values stored in the chart`values.yaml`

Reply:
also, 
>with the actual number of nodes in your cluster

do we want to provide some guidance for cluster that autoscale? I think we'd want that number of nodes to be the maximum number of nodes you'd expect to reach?

Reply:
also,
>Replace `<NUMBER_OF_NODES>` with the actual number of nodes

is `NUMBER_OF_NODES` referring to something in text below, or to the image above? the way this is written makes me think it would be referenced below

Reply:
Thanks for the great feedback!

1. Fixed the placeholder. 
2. Changes all to use a values-override.yml - I think that makes more sense, would you agree?
3. I really wish I could provide better data related to machine sizing, and autoscaling. I also wish I could talk more about steady-state vs pod churn - and include some way to calculate based on typical pod counts... plus ways to find those things. I believe in the end, it will be part of our active monitoring story. So I expect to revisit this doc, it is more of a line in the sand.

Does that seem reasonable?

Reply:
for sure, makes sense 👍